### ISSUE TITLE
Image editing (in place) before uploading

### ISSUE THREAD

**Issue by DmitrySharabin** [mavoweb/plugins#35]
Are there any chances we could edit an image (crop, scale, rotate, etc.) before uploading it to the chosen backend?
I'm planning to develop an app that lets an end user upload images (as far as I know, that function is already available) and make a sequence out of them. So I thought it would be cool if they could make some simple operation with the image right before sending it to the server.
I looked through Github and found [a JavaScript library](https://github.com/carstenschaefer/ImagerJs) that solves the problem as I suppose it to be solved. From my point of view, the library has many cool features (including localization 😊) and well-documented. At the moment, I'm a bit far from writing a plugin for that feature myself (I hope someday I can manage it 😉) so I decided to request it.

**Comment by DmitrySharabin** [comment]
Playing with that library (that is really well made), I've learned that it's not the best choice for the plugin: the library heavily depends on jQuery.
So I did another search and found another (dependency free) library that is as good as I expected it to be—[Cropper.js](https://github.com/fengyuanchen/cropperjs). I started to write the corresponding plugin for Mavo: the new library just works like a charm ☺️. But I still don't have enough knowledge of registering new elements in Mavo (trying to use the analogy with other plugins). Experiments continue! I hope to manage it soon.
